backend/src/main/java/com/jobwait/persistence/PostgresController.java

```diff
@@ -0,0 +1,179 @@
+package com.jobwait.persistence;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.time.OffsetDateTime;
+import java.util.List;
+import java.util.UUID;
+
+import com.jobwait.domain.Answers;
+import com.jobwait.domain.User;
+import com.jobwait.domain.ValidEducationLevel;
+import com.jobwait.domain.ValidWorkContract;
+import com.jobwait.domain.ValidWorkModel;
+import com.jobwait.persistence.adapters.PostgresUserAdapter;
+
+public class PostgresController extends PersistenceController {
+    private String jdbcUrl = "jdbc:postgresql://localhost:5432/mydatabase";
+    private String dbUser = "postgres";
+    private String dbPassword = "password";
+
+    private Connection getConnection() throws SQLException {
+        return DriverManager.getConnection(jdbcUrl, dbUser, dbPassword);
+    }
+
+    @Override
+    public User getUserFromAuthId(String authId) {
+        try {
+            Connection connection = getConnection();
+            PreparedStatement statement = connection.prepareStatement("SELECT * FROM users WHERE authhash = ?");
+            statement.setString(1, authId);
+            ResultSet resultSet = statement.executeQuery();
+            List<User> users = Util.resultSetRowsToAdaptedRows(resultSet, new PostgresUserAdapter());
```

Comment:
would prefer if you didnt call this `Util` but something else, it can be confused with java.util. Perhaps PersistenceUtility?

Reply:
Good point, done.